### PR TITLE
#160617656 write unit tests for all functionality

### PR DESCRIPTION
**Pull request by kcharles52**
**What does this PR do?**

- Write unit tests 

**Description of Task to be completed?**

writing unit tests for ALL functionality that will be implemented by authors heaven

**How should this be manually tested?**

- run the command ` $python3 manage.py test `

**What are the relevant pivotal tracker stories?**

- [#160617656](https://www.pivotaltracker.com/story/show/160617656)

### CODE REVIEW

**Review by nakiwuge — COMMENTED** [review]
Hey Charles kindly squash your commits into one and do the PR again

**Review by PatrickCmd — COMMENTED** [review]
@kcharles52 The naming of your PRs and commits is not acceptable, please do review the engineering guide lines.

**Comment by kcharles52** [comment]
@PatrickCmd and @zellymiriam I have made the necessary changes

File: pytest.ini [hunk 1]
```diff
@@ -0,0 +1,2 @@
+[pytest]
```
Comment (kimbugp) [inline pytest.ini:1]: Does this need to be put up on GitHub?

Comment (kcharles52) [reply pytest.ini:1]: more lines of code are going to be put in there

**Review by PatrickCmd — APPROVED** [review]

**Pull request merged**